modules/nw-aws-load-balancer-with-outposts.adoc

```diff
@@ -1,11 +1,28 @@
 // Module included in the following assemblies:
 // * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+// * post_installation_configuration/configuring-aws-outposts.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-aws-load-balancer-with-outposts_{context}"]
-= AWS Load Balancer Operator and AWS Outposts
+= Using the AWS Load Balancer Operator with AWS Outposts
 
-AWS Load Balancer Operator can work with AWS Outposts. Application Load Balancer is supported. Network Load balancer is not, so AWS Load Balancer Operator does not provision it on AWS Outposts subnets. Ingress resources must be annotated with the either outposts or regular subnets (but not both).
+You can configure the AWS Load Balancer Operator to provision an AWS Application Load Balancer in an Outpost.
+AWS Outposts does not support AWS Network Load Balancers.
+As a result, the AWS Load Balancer Operator cannot provision Network Load Balancers in an Outpost.
+
+You can create an individual AWS Application Load Balancer either in the cloud subnet or in the Outpost subnet, but they must be separate instances.
+An Application Load Balancer in the cloud can attach to cloud-based compute nodes and an Application Load Balancer in the Outpost can attach to edge compute nodes.
+You must annotate Ingress resources with the Outpost subnet or the VPC subnet, but not both.
+
+.Prerequisites
+
+* You have extended an AWS VPC cluster into an Outpost.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
```

Comment:
Why `cluster-admin` permissions? It's not needed for creating an `Ingress`.

Reply:
That was just an assumption on my part, based on many other commands. Will remove it, thanks!